planetiler-custommap/src/test/java/com/onthegomap/planetiler/custommap/ConfiguredSingleFeatureTest.java

```diff
@@ -0,0 +1,127 @@
+package com.onthegomap.planetiler.custommap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import com.onthegomap.planetiler.FeatureCollector;
+import com.onthegomap.planetiler.FeatureCollector.Feature;
+import com.onthegomap.planetiler.Profile;
+import com.onthegomap.planetiler.TestUtils;
+import com.onthegomap.planetiler.config.PlanetilerConfig;
+import com.onthegomap.planetiler.custommap.configschema.SchemaConfig;
+import com.onthegomap.planetiler.reader.SimpleFeature;
+import com.onthegomap.planetiler.reader.SourceFeature;
+import com.onthegomap.planetiler.stats.Stats;
+import java.io.FileInputStream;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.yaml.snakeyaml.Yaml;
+
+public class ConfiguredSingleFeatureTest {
+
+  private static FeatureCollector polygonFeatureCollector() {
+    PlanetilerConfig config = PlanetilerConfig.defaults();
+    FeatureCollector.Factory factory = new FeatureCollector.Factory(config, Stats.inMemory());
+    return factory.get(SimpleFeature.create(TestUtils.newPolygon(0, 0, 0.1, 0, 0.1, 0.1, 0, 0), new HashMap<>()));
+  }
+
+  private static FeatureCollector linestringFeatureCollector() {
+    PlanetilerConfig config = PlanetilerConfig.defaults();
+    FeatureCollector.Factory factory = new FeatureCollector.Factory(config, Stats.inMemory());
+    return factory.get(SimpleFeature.create(TestUtils.newLineString(0, 0, 0.1, 0, 0.1, 0.1, 0, 0), new HashMap<>()));
+  }
+
+  private static Profile configureProfile(String filename) throws Exception {
+    Path staticAttributeConfig = TestConfigurableUtils.pathToResource(filename);
+    Yaml yml = new Yaml();
+    SchemaConfig schema = yml.loadAs(new FileInputStream(staticAttributeConfig.toFile()), SchemaConfig.class);
```

Comment:
In general I try to avoid `File` where possible and use nio utilities instead, for example: `java.nio.file.Files.newInputStream(staticAttributeConfig)`

Reply:
Resolved in f8f0838